docker/package/systemd.py

```diff
@@ -69,8 +69,10 @@ def print_service_file(service_file: ServiceFile, out):
         environment = "".join(
             f'Environment="{x}"\n' for x in service_file.service.environment
         )
-    if service_file.service.environment_file is not None:
-        environment_file = f"EnvironmentFile={service_file.service.environment_file}"
+    if service_file.service.environment_files is not None:
+        environment_files = "\n".join(
+            f'EnvironmentFile={x}' for x in service_file.service.environment_files
```

Comment:
TIL that brackets aren't required for such comprehensions

Reply:
AFAIU it creates generator instead of list, which is also fine for `join`